e2e-tests/functions

```diff
@@ -9,7 +9,8 @@ IMAGE_MONGOD=${IMAGE_MONGOD:-"perconalab/percona-server-mongodb-operator:main-mo
 IMAGE_MONGOD_CHAIN=${IMAGE_MONGOD_CHAIN:-$'
 perconalab/percona-server-mongodb-operator:main-mongod4.4
 perconalab/percona-server-mongodb-operator:main-mongod5.0
-perconalab/percona-server-mongodb-operator:main-mongod6.0'}
+perconalab/percona-server-mongodb-operator:main-mongod6.0
+perconalab/percona-server-mongodb-operator:main-mongod7.0'}
```

Comment:
@tplavcic @inelpandzic I think we need to remove line 10 (psmdb4.4)

Reply:
[d9f27f7](https://github.com/percona/percona-server-mongodb-operator/pull/1470/commits/d9f27f77f0c9ec97ec0e48682075b06be5c48da1)